modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js

```diff
@@ -75,6 +97,15 @@ export default function ChangeTrackingIndicator({
 		navigateUtil(portletURL);
 	};
 
+	const setParameter = useCallback(
+		(url, name, value) => {
+			return (
+				url + '&' + namespace + name + '=' + encodeURIComponent(value)
```

Comment:
Use `createPortletURL` to add the param to the url. You can also call fetch here to simplify the code since the url is always used with fetch. See https://github.com/liferay/liferay-portal/blob/master/modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/PublicationsSearchContainer.js#L120-L134 for example.